README.md

```diff
@@ -18,13 +18,28 @@ We provide a sample [notebook](notebooks/SynthDet_Statistics.ipynb) to help you
 
 Dataset download provides tools to download datasets from HTTP(s), GCS and Unity simulation project . You can run `download` command:
 
-[Download Dataset](https://datasetinsights.readthedocs.io/en/latest/datasetinsights.commands.html#datasetinsights-commands-download)
+[CLI](https://datasetinsights.readthedocs.io/en/latest/datasetinsights.commands.html#datasetinsights-commands-download)
 
 ```bash
 datasetinsights download \
   --source-uri=<xxx> \
   --output=$HOME/data
 ```
+[Programmatically](https://datasetinsights.readthedocs.io/en/latest/datasetinsights.io.downloader.html#module-datasetinsights.io.downloader.gcs_downloader)
```

Comment:
Do we need to add this instruction here?

Reply:
Yes, Please refer release checklist: https://github.com/Unity-Technologies/datasetinsights/issues/143
"Update README to reflect the current high level API."